test/BUILD.gn

```diff
@@ -175,8 +176,6 @@ static_library("browser_test_support") {
   sources = [
     "base/brave_test_launcher_delegate.cc",
     "base/brave_test_launcher_delegate.h",
-    "//brave/app/brave_main_delegate.cc",
-    "//brave/app/brave_main_delegate.h",
```

Comment:
These aren't needed here anymore since they are included anywhere that the chrome main delegate is because of the chromium_src override.